cpp/vcpkg.json

```diff
@@ -109,7 +109,7 @@
   "overrides": [
     { "name": "openssl", "version-string": "3.3.0" },
     { "name": "arrow", "version": "18.0.0" },
-    { "name": "aws-sdk-cpp", "version": "1.11.474" },
+    { "name": "aws-sdk-cpp", "version": "1.11.474", "$note": "Update overlay json to upgrade" },
```

Comment:
Can you put a note here warning people not to upgrade to 1.11.486+ yet?

https://github.com/aws/aws-sdk-cpp/issues/3253